src/Microsoft.ML.UniversalModelFormat/Onnx/OnnxMl.md

```diff
@@ -4,11 +4,11 @@
 2. Download protobuf C# compiler version 3.0 or greater from 
    https://github.com/google/protobuf/tree/master/csharp
 3. Add `option csharp_namespace =
-   "Microsoft.MachineLearning.Runtime.UniversalModelFormat.Onnx";` to `onnx-ml.proto3` right below `package ONNX_NAMESPACE;`
+   "Microsoft.ML.Runtime.UniversalModelFormat.Onnx";` to `onnx-ml.proto3` right below `package ONNX_NAMESPACE;`
 4. Assuming the compiler and proto file are saved at
    `E:\protobuf-csharp-port\lib` then run the following in command line to get C# code from the proto file:
    ```
    protoc.exe -I="E:\protobuf-csharp-port\lib" --csharp_out="E:\protobuf-csharp-port\lib" "E:\protobuf-csharp-port\lib\onnx-ml.proto3"
    ```
 
-## The proto3 file is current as of 02/07/2018.
\ No newline at end of file
+## The proto3 file is current as of 06/01/2018 and generated from onnx-ml.proto3 based on https://github.com/onnx/onnx/pull/1052
```

Comment:
>based on https://github.com/onnx/onnx/pull/1052 [](start = 81, length = 47)

This hasn't been checked into their master, but is in code currently under review?

Reply:
@TomFinley This was checked-into master before I used it... 	linkerzhang merged 7 commits into onnx:master from linkerzhang:kezhan/proto3_fix 5 hours ago, it says on that link.

Reply:
Great. In future always use a commit ID.

---
In reply to: [192512640](https://github.com/dotnet/machinelearning/pull/248#discussion_r192512640) [](ancestors = 192512640)